device-types/Fortinet/FG-120G.yaml

```diff
@@ -0,0 +1,82 @@
+---
+manufacturer: Fortinet
+model: FortiGate 120G
+slug: fortinet-fg-120g
+part_number: FG-120G
+u_height: 1
+is_full_depth: false
+airflow: side-to-rear
+comments: '[FortiGate 120G Series Datasheet](https://www.fortinet.com/content/dam/fortinet/assets/data-sheets/fortigate-120g-series.pdf)'
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 40
+    allocated_draw: 38
+  - name: PS2
+    type: iec-60320-c14
+    maximum_draw: 40
+    allocated_draw: 38
+interfaces:
+  - name: ha
+    type: 1000base-t
+  - name: mgmt
+    type: 1000base-t
```

Comment:
Are either of these out of band management interfaces? If so, you should add `mgmt_only: true`